llvm/lib/IR/RepoDefinition.cpp

```diff
@@ -169,29 +169,42 @@ void ModuleHashGenerator::calculateGONumAndGOIMap(const Module &M) {
 #endif
 }
 
-void ModuleHashGenerator::updateDigestUseContributions(
-    MD5 &GOHash, const GOVec &Contributions) {
-  for (const GlobalObject *const G : Contributions) {
-    GOHash.update(GOIMap[G].InitialDigest.Bytes);
-  }
+size_t ModuleHashGenerator::updateDigestUseContributions(
+    const GlobalObject *GO, MD5 &GOHash, const GOVec &Contributions,
+    bool UseRepoDefinitionMD) {
   // GOHash/Module is changed if Contributions is not empty.
   Changed = Changed || !Contributions.empty();
```

Comment:
Does this apply to dependencies as well?

Reply:
Good point. Added it and removed unnecessary code.